epaint/benches/benchmark.rs

```diff
@@ -0,0 +1,40 @@
+#![feature(test)]
```

Comment:
This is an unstable feature - either use [Criterion](https://crates.io/crates/criterion) (which `egui_demo_lib` uses), or if that's too much headache, just remove the benchmarks again :)

Reply:
Oh, okay. I considered using criterion but was not sure. I will do that then.